service-front/module/Application/src/Helpers/LpaFormHelper.php

```diff
@@ -0,0 +1,263 @@
+                    'address' => $idCheck['address'],
+                    'name_match' => $idCheck['name_match'],
+                    'address_match' => $idCheck['address_match'],
+                    'error' => $idCheck['error']
+                ];
+                $result['data'] = [
+                    "case_uuid" => $uuid,
+                    "LPA_Number" => $form->get('lpa')->getValue(),
+                    "Type_Of_LPA" => $this->getLpaTypeFromSiriusResponse($siriusCheck),
+                    "Donor" => $this->getDonorNameFromSiriusResponse($siriusCheck),
+                    "Status" => $statusCheck['status'],
+                    "CP_Name" => $detailsData['firstName'] . " " . $detailsData['lastName'],
+                    "CP_Address" => $detailsData['address']
+                ];
+            } elseif ($channelCheck['error'] === true) {
+                $result['status'] = $channelCheck['status'];
+                $result['message'] = $channelCheck['message'];
+                $result['data'] = [
+                    "case_uuid" => $uuid,
+                    "LPA_Number" => $form->get('lpa')->getValue(),
+                    "Type_Of_LPA" => $this->getLpaTypeFromSiriusResponse($siriusCheck),
+                    "Donor" => $this->getDonorNameFromSiriusResponse($siriusCheck),
+                    "Status" => $statusCheck['status'],
+                    "CP_Name" => $detailsData['firstName'] . " " . $detailsData['lastName'],
+                    "CP_Address" => $detailsData['address']
+                ];
+            } else {
+                $result['status'] = "Success";
+                $result['message'] = "";
+                $result['data'] = [
+                    "case_uuid" => $uuid,
+                    "LPA_Number" => $form->get('lpa')->getValue(),
+                    "Type_Of_LPA" => $this->getLpaTypeFromSiriusResponse($siriusCheck),
+                    "Donor" => $this->getDonorNameFromSiriusResponse($siriusCheck),
+                    "Status" => $statusCheck['status'],
+                    "CP_Name" => $detailsData['firstName'] . " " . $detailsData['lastName'],
+                    "CP_Address" => $detailsData['address']
+                ];
+            }
+        }
+
+        return new LpaFormHelperResponseDto(
+            $uuid,
+            $form,
+            [
+                'lpa_response' => $result
+            ],
+        );
+    }
+
+    public function getCpAddressFromSiriusResponse(array $siriusCheck): array
+    {
+        try {
+            return $siriusCheck['opg.poas.lpastore']['certificateProvider']['address'];
+        } catch (\Exception $exception) {
+            return [$exception->getMessage()];
+        }
+    }
+
+    public function getLpaTypeFromSiriusResponse(array $siriusCheck): string
+    {
+        try {
+            return $siriusCheck['opg.poas.lpastore']['lpaType'];
+        } catch (\Exception $exception) {
+            return $exception->getMessage();
+        }
```

Comment:
The only error this is likely to throw is a PHP Warning if the key doesn't exist, and PHP Warnings can't be caught with a try..catch.

Instead, you probably want to do something like:
```suggestion
        return $siriusCheck['opg.poas.lpastore']['lpaType'] ?? '';
```
which will return an empty string if the key didn't exist.